src/main/java/com/google/sps/data/UserInfo.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class UserInfo {
+
+  public static final String DATA_TYPE = "UserInfo";
+  public static final String ID = "userId";
+  public static final String NICKNAME = "nickname";
+  public static final String CREATED_EVENTS = "created_events";
+  public static final String BOOKMARKED_EVENTS = "bookmarked_events";
+  public static final String CURRENT_CHALLENGE = "current_challenge";
+  public static final String CHALLENGE_STATUSES = "challenge_statuses";
```

Comment:
I think it would be best to move these constants to UserServlet since they don't provide much functionality to this class.

Reply:
Also ended up using them in EventsServlet so I think I'm going to leave it as is to avoid duplicating code